kafka/integration_test.go

```diff
@@ -1420,6 +1420,7 @@ func (its *IntegrationTestSuite) TestAdminClient_DescribeTopics() {
 		"Expected correct error for nonexistent topic")
 
 	topicDesc := topicDescs[0]
+	assert.NotNil(topicDesc.TopicID)
```

Comment:
assert.NotNil asways returns true for structs according to this issue https://github.com/stretchr/testify/issues/570
Suggest:
```suggestion
	assert.NotZero(topicDesc.TopicID.GetLeastSignificantBits())
	assert.NotZero(topicDesc.TopicID.GetMostSignificantBits())
	assert.NotEmpty(topicDesc.TopicID.String())
```